src/briefcase/platforms/macOS/app.py

```diff
@@ -25,10 +22,25 @@ def binary_path(self, app):
     def distribution_path(self, app):
         return self.binary_path(app)
 
+    def entitlements_path(self, app):
+        return (
+                self.bundle_path(app)
+                / 'Entitlements.plist'
+        )
+
 
 class macOSAppCreateCommand(macOSAppMixin, CreateCommand):
     description = "Create and populate a macOS app."
 
+    def install_app_support_package(self, app: BaseConfig):
+        """
+        Install the application support packge.
+
+        :param app: The config object for the app
+        """
+        # nothing to do here template already contains built binary
```

Comment:
I see you've put the standard library in the template; we will want to avoid that, as it couples the template to a specific Python version.

Reply:
>  we will want to avoid that, as it couples the template to a specific Python version.

Do we want to avoid that? The template is already firmly coupled to the Python version, considering that the binary contains a statically linked Python interpreter. Or would you rather move the binary into a custom "app" support package? 

This is probably one more reason to move to a framework build. I wish I had more experience with that but this is really above my pay grade...